extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -988,7 +992,10 @@ export class AmpA4A extends AMP.BaseElement {
           // modified url.
           'src': xhrFor(this.win).getCorsUrl(this.win, adUrl),
         }, SHARED_IFRAME_PROPERTIES));
-    return this.iframeRenderHelper_(iframe);
+    const sentinel = generateSentinel(window);
+    const metadata = getContextMetadata(window, iframe, sentinel);
+    iframe.setAttribute('name', encodeURI(JSON.stringify(metadata)));
```

Comment:
I just tested it in js console and setAttribute did not escape anything. will change to encodeURIComponent